pkgs/by-name/au/auto-editor/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  substituteAll,
+  ffmpeg,
+  yt-dlp,
+}:
+
+let
+  inherit (python3Packages)
+    buildPythonApplication
+    setuptools
+    numpy
+    av
+    ;
+in
+buildPythonApplication rec {
+  pname = "auto-editor";
+  version = "24w19a";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "WyattBlue";
+    repo = "auto-editor";
+    rev = "refs/tags/${version}";
+    hash = "sha256-dXiNU9wPubmlZJesxvQYGvv1ga68wt/rsP6L2sWtYkE=";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./set-exe-paths.patch;
+      ffmpeg_path = lib.getExe ffmpeg;
+      yt_dlp_path = lib.getExe yt-dlp;
+    })
+  ];
+
+  postPatch = ''
+    # pyav is a fork of av, but has since been un-forked
+    substituteInPlace pyproject.toml \
```

Comment:
We can use `pythonRelaxDepsHook` in this case.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook